### DISCUSSION TITLE
buddhism/confusion/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#23] [General]
# buddhism/confusion/

https://shanenull.com/buddhism/confusion/

**Comment by shane0** [comment]
**Jay Garfield's explanation of primordial confusion in the style of Cormac McCarthy:**

Primordial confusion is the fundamental ignorance that is the root of all suffering.

It is the belief that we are separate from the world around us.

It is the illusion of self.

Primordial confusion is like a cloud that obscures our true nature.

It prevents us from seeing the world as it really is.

It prevents us from seeing our own true nature.

Jay Garfield explains primordial confusion as the failure to recognize that our experience of the world is mediated by our own minds.

We mistake our thoughts and perceptions for reality.

We believe that the world is as we see it, and that we are separate from it.

But this is not the case.

The world is not as we see it.

And we are not separate from it.

Primordial confusion is the source of all our problems.

It is the source of our attachment, our aversion, and our ignorance.

It is the source of our suffering.

But there is hope.

We can overcome primordial confusion through the practice of meditation.

Through meditation, we can learn to see the world as it really is.

Through meditation, we can learn to see our own true nature.

Through meditation, we can find freedom from suffering.

So sit down and meditate.

And let the light of awareness dispel the cloud of primordial confusion.

**Note:** I have taken some liberties with Garfield's explanation in order to fit the style of Cormac McCarthy.
